ush/build.sh

```diff
@@ -6,6 +6,7 @@ set -eux
 readonly DIR_ROOT=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.." && pwd -P)
 
 # User Options
+# BUILD_TYPE=${BUILD_TYPE:-"Debug"}
```

Comment:
Was this unintended?

Reply:
The line is commented out, but should be removed.  I was testing the code in debug mode as well!